plugins/adminhelp.sma

```diff
@@ -13,133 +13,181 @@
 
 #include <amxmodx>
 
-new g_cvarDisplayClientMessage;
-new g_cvarHelpAmount;
+const MaxMapLength         = 32;
```

Comment:
If you take a look around ReHLDS, it looks like it is indeed 32. If you find proof that it's 64, link it.